realm/realm-jni/src/io_realm_internal_LinkView.cpp

```diff
@@ -178,8 +177,7 @@ JNIEXPORT jlong JNICALL Java_io_realm_internal_LinkView_nativeWhere
     try {
         LinkViewRef *lv = LV(nativeLinkViewPtr);
         LinkViewRef lvr = *lv;
-        Query query = lvr->get_target_table().where(LinkViewRef(lvr));
-        TableQuery* queryPtr = new TableQuery(query);
+        Query *queryPtr = new Query(std::move(lvr->get_target_table().where(LinkViewRef(lvr))));
```

Comment:
The use of `std::move` here is unnecessary since `lvr->get_target_table().where(LinkViewRef(lvd))` is already an rvalue.